src/eva/transforms/channel_stats.py

```diff
@@ -0,0 +1,62 @@
+# (C) Copyright 2021-2022 NOAA/NWS/EMC
+#
+# (C) Copyright 2021-2022 United States Government as represented by the Administrator of the
+# National Aeronautics and Space Administration. All Rights Reserved.
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+
+# --------------------------------------------------------------------------------------------------
+
+
+from math import sqrt
+import re
+from statistics import mean
+
+from eva.utilities.config import get
+from eva.utilities.logger import Logger
+from eva.transforms.transform_utils import parse_for_dict, split_collectiongroupvariable
+from eva.transforms.transform_utils import replace_cgv
+from eva.utilities.utils import remove_list_duplicates
+from eva.utilities.utils import remove_empty_from_list_of_strings
+
+
+# --------------------------------------------------------------------------------------------------
+
+
+def channel_stats(config, data_collections):
+
+    # Create a logger
+    logger = Logger('Stats2DTransform')
+
+    # Define stat functions to calcualte
+    stat_functions = ['Mean', 'Std', 'Count', 'Median', 'Min', 'Max']
```

Comment:
Do we ever want to specify the list of stats or always compute these 6?

Reply:
@CoryMartin-NOAA I thought this was the most user-friendly way of going about it.   But are you suggesting I add the capability to specify stat_functions in the YAML with this as a default?

Reply:
That is my suggestion but not a requirement if you don't think it is necessary.